[Gardening] Event timing tests no longer flaky

francovs opened the pull request:
#### 365b741dcfa87445af334b02ff96b716bb4994c2
<pre>
[Gardening] Event timing tests no longer flaky
<a href="https://bugs.webkit.org/show_bug.cgi?id=298768">https://bugs.webkit.org/show_bug.cgi?id=298768</a>
<a href="https://rdar.apple.com/problem/160454306">rdar://problem/160454306</a>

Reviewed by Ryan Reno.

After converting to Monotonic timestamps (<a href="https://commits.webkit.org/49980@main)">https://commits.webkit.org/49980@main)</a>
and re-importing from WPT (<a href="https://commits.webkit.org/300206@main)">https://commits.webkit.org/300206@main)</a>, some
event timing tests should no longer be flaky.

Canonical link: <a href="https://commits.webkit.org/300482@main">https://commits.webkit.org/300482@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2c00eb3d2b6518ec5bc8010444107b4591632b80

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/122204 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/41907 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/32576 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/128778 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/74298 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/42621 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/50500 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/92881 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/61745 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/125156 "Passed tests") | [⏳ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-WK2-Tests-EWS "Waiting to run tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/109432 "Passed tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/73536 "Found 1 new API test failure: WPE/TestWebKitWebContext:/webkit/WebKitWebContext/no-web-process-leak (failure)") | 
| | [⏳ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-WPT-WK2-Tests-EWS "Waiting to run tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/27594 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/72268 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/103505 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/27788 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/131528 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/49143 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/37387 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/101449 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/49520 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/105643 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/101319 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25815 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/46694 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/24809 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/45894 "The change is no longer eligible for processing.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/49000 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/48470 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/51820 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/50150 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/e61d43621ee998479ff9bb65e133d463d90e4de3. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50650

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/af049c90600700e5a1e951fc815ef3fc35126f86. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50650

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/26edefd802c525f0ac6f98fc2dd2967147a938b9. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50650

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/2c00eb3d2b6518ec5bc8010444107b4591632b80. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50650

francovs committed:
[Gardening] Event timing tests no longer flaky
https://bugs.webkit.org/show_bug.cgi?id=298768
rdar://problem/160454306

Reviewed by Ryan Reno.

After converting to Monotonic timestamps (https://commits.webkit.org/49980@main)
and re-importing from WPT (https://commits.webkit.org/300206@main), some
event timing tests should no longer be flaky.

Canonical link: https://commits.webkit.org/300482@main
